lua/nvim-treesitter/indent.lua

```diff
@@ -22,20 +22,21 @@ local function node_fmt(node)
 end
 
 local get_indents = utils.memoize_by_buf_tick(function(bufnr)
-  local indents = queries.get_capture_matches(bufnr, '@indent.node', 'indents') or {}
-  local branches = queries.get_capture_matches(bufnr, '@branch.node', 'indents') or {}
-
-  local indents_map = {}
-  for _, node in ipairs(indents) do
-    indents_map[tostring(node)] = true
-  end
-
-  local branches_map = {}
-  for _, node in ipairs(branches) do
-    branches_map[tostring(node)] = true
+  local get_map = function(capture)
+    local matches = queries.get_capture_matches(bufnr, capture, 'indents') or {}
+    local map = {}
+    for _, node in ipairs(matches) do
+      map[tostring(node)] = true
```

Comment:
I'd rather use `node:id()` here :
```suggestion
      map[node:id()] = true
```